src-back/app/fight/fight_session.py

```diff
@@ -27,7 +28,7 @@ def __init__(self, fight_id, player_list):
         self._players_sessions = {}
         self._players_typing_history = {player: [] for player in player_list}
         self._scores = {player: 0 for player in player_list}
-        self._words_to_find = ["word", "broccoli", "tomato", "bee", "beekeeper"]
+        self._words_to_find = get_random_word_list(amount=100)
```

Comment:
Just wanted to make sure : We will have to make this configurable later, since the "Lobby" page dictates what categories are being selected.
So this info will have to pass through one of the routes and end up here. If that is ok, you can discard this comment.

Reply:
I would assume that the constructor takes additional parameters that it forwards to the random function. If the function do not fulfill the requirements it just has to be modified and take necessary parameters, and implement necessary logic.